numba/cpython/tupleobj.py

```diff
@@ -119,9 +127,10 @@ def unituple_constant(context, builder, ty, pyval):
         context, builder, ty, cgutils.pack_array(builder, consts),
     )
 
+
 @lower_constant(types.Tuple)
 @lower_constant(types.NamedTuple)
-def unituple_constant(context, builder, ty, pyval):
+def unituple_constant_(context, builder, ty, pyval):
```

Comment:
Why has this gained an underscore at the end?

Reply:
flake8, I believe